src/core/tooltip.py

```diff
@@ -0,0 +1,67 @@
+""" tk_ToolTip_class101.py
```

Comment:
Let's update this first line to match the name of the file in this project, just for clarify. The rest of the attribution can remain as-in.